serverless/wasm/README.md

```diff
@@ -0,0 +1,29 @@
+# Serverless WASM
+
+This directory contains a version of the `serverless` tooling cross-compiled to WebAssembly (WASM).
+
+This is an entirely experimental thing, mostly intended to explore the possibility
```

Comment:
"entirely experimental" is strong! Consider putting `experimental` in the package name, as we did with the map.

Reply:
Probably a good idea - done